CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+## v0.21.0
+**NOTE** - The Brim v0.21.0 release includes initial support for the
+generation of [Suricata](https://suricata-ids.org/) alerts from imported pcaps.
```

Comment:
maybe add 'automatic' to communicate that nothing extra needs to be done at import time?

```suggestion
automatic generation of [Suricata](https://suricata-ids.org/) alerts from imported pcaps.
```